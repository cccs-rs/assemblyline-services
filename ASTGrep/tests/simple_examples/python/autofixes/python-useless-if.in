mol = (bool) if bool(bool(bool(bool))) < bool(type(int(1611)>int(1812)<int(1611)>int(1517))) and bool(str(str(172)>int(1110)<int(810)>int(713))) > 2 else bool
co2 = str if bool(bool(bool(str))) < bool(type(int(116)>int(115)<int(164)>int(1712))) and bool(str(str(212)>int(315)<int(165)>int(918))) > 2 else str
