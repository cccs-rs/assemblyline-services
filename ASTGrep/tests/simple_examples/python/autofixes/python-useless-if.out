mol = bool
co2 = str
